Jonah/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<head>
+      <meta charset="utf-8">
+      <title>Your joke page</title>
+      <link rel="stylesheet" href="stylesheet.css">
+      <link rel="preconnect" href="https://fonts.googleapis.com">
+      <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+      <link href="https://fonts.googleapis.com/css2?family=Estonia&display=swap" rel="stylesheet">
+  </head>
+
+  <body class="background">
+    <h1>Tell me a joke</h1>
+    <main>
+      <div>
+        <p>Knock knock</p>
+      </div>
+      <div>
+        <em>Who's there?</em>
+      </div>
+      <small>Mikel WUZ HERE!!</small>
+    
+      <div id="Chinese">
+      一个男人告诉他的医生，“医生，帮帮我。我沉迷于推特！医生回答说：“对不起，我没有跟着你
+    </div >
+    <div id="English">
+    <p> <strong> tells his doctor, “Doc, help me. I’m addicted to Twitter! The doctor replies, “Sorry, I don’t follow you</strong> </p>
+    </div>
+    <button onclick="showOrHideDiv()">Translate Punchline</button>
+    <script>
+   
+      function showOrHideDiv() {
+            if (document.getElementById('Chinese')) {
+
+                if (document.getElementById('Chinese').style.display == 'none') {
```

Comment:
It's best to use the strict equality `===` operator when comparing values

```suggestion
                if (document.getElementById('Chinese').style.display === 'none') {
```